vagrant/provisioning/roles/solr/templates/solr.service

```diff
@@ -1,17 +1,12 @@
 [Unit]
-Description=Solr service
+Description=SOLR Service
+
 
 [Service]
 Type=forking
-RemainAfterExit=true
-ExecStart={{ root_folder }}/app/solr/bin/init.d/solr start
-ExecStop={{ root_folder }}/app/solr/bin/init.d/solr stop
-# Starting from CentOS 7.7 we can't specify the PIDfile for 
-# solr anymore; but systemd can find it from the process 
-# table anyway.
-# PIDFile={{ root_folder }}/app/solr/bin/solr-8983.pid
 User=root
```

Comment:
I think the user should `solr`, not `root`

Reply:
Somehow solr requires to start as root user, then it forks a child process as the user indicated in the solr.in.sh file.

Reply:
Changed the user to solr as well

Reply:
@david-oc-miller When I tested that with @dzdimov , I actually had to set the user to `solr` in the unit file. @dzdimov Have you tested it with the `solr` user?

Reply:
Well, maybe the requirement to start as root was imposed by the /etc/init.d-style script.  Now that we call solr directly you probably do have to run as solr user.  This revised service unit is a huge improvement over the old one!